### PR TITLE
chore: turns google doc urls into links

### PR DESCRIPTION
**Pull request by jhackett1**
satisfies https://trello.com/c/vJzhNSpb/910-link-text-to-google-docs-in-case-note-popup-should-be-actual-links

### CODE REVIEW

File: components/ResidentPage/CaseNoteDialog.tsx [hunk 1]
```diff
@@ -56,6 +56,19 @@ interface HistoricContentProps {
 const prettyKey = (key: string): string =>
   key?.replace(/_/g, ' ')?.replace(/^\w/, (char) => char.toUpperCase());
 
+const PrettyValue = ({ value }: { value: string }): React.ReactElement => {
+  if (
+    // detect urls
+    // eslint-disable-next-line no-useless-escape
+    /((?:(?:http?|ftp)[s]*:\/\/)?[a-z0-9-%\/\&=?\.]+\.[a-z]{2,4}\/?([^\s<>\#%"\,\{\}\\|\\\^\[\]`]+)?)/gi.test(
```
Comment (AlexHerbertMadeTech) [inline components/ResidentPage/CaseNoteDialog.tsx:63]: Not sure about this regex, I see FTP in there. Will this turn every case note link into a clickable?

Comment (jhackett1) [reply components/ResidentPage/CaseNoteDialog.tsx:63]: i copied it from a highly starred stackoverflow answer. it's supposed to turn anything in the case form data that could be a url into one

Comment (AlexHerbertMadeTech) [reply components/ResidentPage/CaseNoteDialog.tsx:63]: Yes, but is turning _any_ URL into something clickable the best th ing for this? As a security minded person, I have a feeling that it's a bad idea to allow users to put whatever in here and also make any link clickabl;e without caring what it's linking to, especially when there are a lot of less computer literate users about.

**Review by AlexHerbertMadeTech — APPROVED** [review]
LGTM

**Pull request merged**